src/components/discover-sheet/UniswapPoolsSection.js

```diff
@@ -184,6 +196,46 @@ export default function UniswapPools({
     [colors, sortDirection]
   );
 
+  const { push } = useNavigation();
+  const removeNextToLastRoute = useRemoveNextToLast();
+  const { nativeCurrency } = useAccountSettings();
+  const genericAssets = useSelector(
+    ({ data: { genericAssets } }) => genericAssets
+  );
+  const { uniswap } = useSelector(readableUniswapSelector);
+
+  const handleOpenExpandedState = useLatestCallback(
+    item => {
+      let poolAsset = uniswap.find(pool => pool.address === item.address);
+      if (!poolAsset) {
+        poolAsset = parseAssetNative(
+          { ...item, ...genericAssets[item.address] },
+          nativeCurrency
+        );
+      }
+
+      analytics.track('Pressed Pools Item', {
+        category: 'discover',
+        symbol: poolAsset.tokenNames,
+        type: item.attribute,
+      });
+
+      // on iOS we handle this on native side
+      android && removeNextToLastRoute();
+
+      push(Routes.EXPANDED_ASSET_SHEET_POOLS, {
+        asset: poolAsset,
+        dpi: true,
+        fromDiscover: true,
+        longFormHeight: initialLiquidityPoolExpandedStateSheetHeight,
+        type: 'uniswap',
+      });
+    },
+    [genericAssets, nativeCurrency, push, removeNextToLastRoute, uniswap]
```

Comment:
This hook doesn't have dependencies

Reply:
Did you check for ts errors?

Reply:
I wasn't looking specifically for TS errors and nothing caught my attention. Probably because it's a JS file so IDE didn't complain?

Fixing anyway. Thanks for catching this!

Reply:
Oh I just noticed that this is a js file lol, sorry.